openstack/compute/v2/extensions/resetstate/doc.go

```diff
@@ -0,0 +1 @@
+package resetstate
```

Comment:
Would you be able to add some extra documentation here? See [this](https://github.com/gophercloud/gophercloud/blob/master/openstack/compute/v2/extensions/startstop/doc.go) as an example.

Reply:
@jtopjian Added some extra documentation.